utils/common-utils/src/jvmMain/kotlin/io/kinference/utils/CommonDataLoader.kt

```diff
@@ -0,0 +1,18 @@
+package io.kinference.utils
+
+import okio.FileSystem
+import okio.Path
+
+actual object CommonDataLoader : DataLoader {
+    actual override suspend fun bytes(path: Path): ByteArray {
+        FileSystem.SYSTEM.read(path) {
```

Comment:
Why do you we need okio? Does it provide anything except Path class that is being used?